### ISSUE TITLE
Jetpack Compose usage

### ISSUE THREAD

**Issue by Fr3di** [tomtom-international/tomtom-navigation-android-examples#100]
Is it possible to use the map (map-display) with Jetpack Compose? 
I am struggling to display the map without using xml

**Comment by Czeach** [comment]
Yes @Fr3di , it's possible. I've written about it [here](https://medium.com/@ezichukwuamarachi/simplifying-tomtom-map-integration-with-a-jetpack-compose-wrapper-b4dcbed22e05)

**Comment by Fr3di** [comment]
Thx but it still requires to use xml file which ideally I would like to avoid after moving fully to Jetpack Compose. Unfortunatelly seems like that's the only way for now. Thx anyway @Czeach :)

**Comment by Czeach** [comment]
You can actually do it without creating the xml container, just replace the container id with `R.id.content` when creating the map fragment. It'll work but it's less ideal.
